<launch>


    <!-- <node pkg="strokeRehabSystem" type="hid2udp.sh" name="hid2udp" output="screen">
  	</node> -->

    <node pkg="strokeRehabSystem" type="udp_node.py" name="UDP" output="screen">
    </node>

</launch>
